tarok/src/cards.cpp

```diff
@@ -84,4 +89,31 @@ std::array<Card, 54> BuildDeck() {
   return deck;
 }
 
+std::tuple<std::vector<int>, std::array<std::vector<int>, 3>> DealCards(
+    int seed = 42) {
```

Comment:
As stated above, I would basically move the default value for `seed` to the state constructor (it could be passed in by the caller as game parameters right?).